src/features/content-explorer/content-explorer/ContentExplorer.js

```diff
@@ -552,6 +559,7 @@ class ContentExplorer extends Component {
                 }}
                 {...contentExplorerProps}
             >
+                {isInfoNoticeVisible && <ContentExplorerInfoNotice infoNoticeText={infoNoticeText} />}
```

Comment:
you can use one prop to control this instead of two:
```suggestion
                {infoNoticeText && <ContentExplorerInfoNotice infoNoticeText={infoNoticeText} />}
```

Reply:
Updated